src/dir.c

```diff
@@ -181,6 +181,29 @@ mrb_dir_chdir(mrb_state *mrb, mrb_value klass)
 }
 
 mrb_value
+mrb_dir_chroot(mrb_state *mrb, mrb_value self)
+{
+  #if defined(_WIN32) || defined(_WIN64) || defined(__android__)
+  mrb_raise(mrb, E_NOTIMP_ERROR, "chroot() unreliable on Win platforms");
+  return mrb_fixnum_value(0);
+  #else
+  mrb_value spath;
+  char *path;
+  int res;
+
+  mrb_get_args(mrb, "S", &spath);
+  path = mrb_str_to_cstr(mrb, spath);
+  res = chroot(path);
+  if (res == -1) {
+    perror("chroot");
```

Comment:
Yeah, I understood this is not desirable(because perror output is uncontrollable via mruby).